src/Indodana.php

```diff
@@ -93,23 +93,35 @@ public function checkout(array $input = []) {
     return $responseJson;
   }
 
+  public function getBaseUrl()
+  {
+    return $this->baseUrl;
+  }
+
+  public function getAuthToken()
+  {
+    return IndodanaSecurity::generateBearerToken(
+      $this->apiKey,
+      $this->apiSecret
+    );
+  }
 
-  public function validateAccessToken($accessToken)
+  public function validateAuthCredentials($credentials)
```

Comment:
Maybe `$credentialString` is better

Reply:
Alright thanks!

Reply:
Btw, I decide to keep it as `credentials` based on reference from https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization